tools/dearimgui_update.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+if [ $# == 0 ] ; then
+    echo "Usage: ./dearimgui_update.sh SOURCE DEST"
+    exit 1;
+fi
+
+dearimgui_path=$1
+libcinder_path=$2
+
+target_inc=$libcinder_path/include/imgui
+target_src=$libcinder_path/src/imgui
+
+cd $dearimgui_path
+cp -v *.h $target_inc; cp -v *.cpp $target_src
+cp -v misc/cpp/*.h $target_inc; cp -v misc/cpp/*.cpp $target_src
+cp -v misc/freetype/*.h $target_inc; cp -v misc/freetype/*.cpp $target_src
+cp -v examples/imgui_impl_opengl3.h $target_inc; cp -v examples/imgui_impl_opengl3.cpp $target_src
```

Comment:
This script should allow easy updates via: `./dearimgui_update.sh ~/Code/imgui ~/Code/Cinder` A flat hierarchy on the cinder-side was preserved to prevent additional include paths. Furthermore, although we had discussed hiding/privatizing implementation files like `imgui_impl_opengl3.h/cpp` into `src\imgui`, I but I kept in public to allow uses cases where the automatic rendering is disabled.